internal/handlers/github.go

```diff
@@ -90,7 +89,8 @@ func (handler GitHubHandler) AuthenticateUser() (string, error) {
 		return "", err
 	}
 
-	if err = exec.Command("open", "https://github.com/login/device").Start(); err != nil {
+	if err = pkg.OpenBrowser("https://github.com/login/device"); err != nil {
+		log.Error().Msgf("error opening browser: %s", err)
 		return "", err
```

Comment:
Will this error fail the installer?

Reply:
For containers run, there is not "browser".. so it would prevent container runs or machine without X if this breaks the installer, if not a error that would fail the installer. I see no issues with it.

Reply:
the github device validation needs a browser to issue a token, containers users needs to pass the token via configuration

Reply:
the point is that if tis fails, I can test device token from container. I can access the link, but I can't open the browser. 

If this break the install the missing browser, we should avoid. As it would work, I used before without token. We are failing for no good reason the installer.

Reply:
I actually didnt add this into this PR, I just added the log.

let's move this discussion into a different topic since it's not the PR goal https://github.com/kubefirst/kubefirst/discussions/1076